src/cfnlint/rules/resources/ApproachingLimitName.py

```diff
@@ -8,7 +8,7 @@
 
 class LimitName(CloudFormationLintRule):
     """Check maximum Resource name size limit"""
-    id = 'I3011'
+    id = 'I3012'
```

Comment:
noticed this but wasn't sure how changing existing rule IDs would work

luckily, this should be one of the rarer rules, so hopefully there aren't many existing `--ignore-checks` for it

Reply:
I figured this one was less used than the other `I3011` rule